### DISCUSSION TITLE
More ability to panels

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1808] [Q&A]
1) Can user change order of panels ?
2) Can user switch off some panels ?
3) Why user cannot decrease panel width - when some panels are on the same side?

**Comment by 20NE** [comment]
3 move journal panel to the other side, then you can decrease panel width

**Comment by Nick222** [comment]
Move to Discussion, pls...

**Reply by introt** [reply]
Done as requested!

**Comment by introt** [comment]
1. Yes, you can rearrange the tabs by dragging from their titles; use plugin settings to move a panel from one side to the other (or to the bottom)
2. Yes, disable the plugins responsible for said panels
3. Currently, the minimum width for a panel is at least the sum of its tabs' titles
